salt/utils/parsers.py

```diff
@@ -1044,12 +1037,13 @@ def start(self):
         self._install_signal_handlers()
 
     def _handle_signals(self, signum, sigframe):  # pylint: disable=unused-argument
-        msg = self.__class__.__name__
+        msg = [six.text_type(self.__class__.__name__)]
         if signum == signal.SIGINT:
-            msg += ' received a SIGINT.'
+            msg.append('received a SIGINT.')
         elif signum == signal.SIGTERM:
-            msg += ' received a SIGTERM.'
-        logging.getLogger(__name__).warning('%s Exiting.', msg)
+            msg.append('received a SIGTERM.')
```

Comment:
Same thing here, this line needs to be reverted.

Reply:
It is earlier an array... :wink: